marsappspa/src/components/ClickCounter.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from "react";
+
+export const ClickCounter: React.FC = () => {
+    const [clickCount, setClickCount] = useState(0);
+
+    function incrementCounter() {
+        setClickCount(clickCount + 1);
+    }
+
+    useEffect(() => {
+        if(clickCount === 0){
+            setClickCount(parseInt(localStorage.getItem("clickCount") || "0"));
+        }
+
+        localStorage.setItem("clickCount", clickCount.toString());
+    },[clickCount]);
```

Comment:
Not that sure this is the proper way to do this, but it works.

Reply:
This is as good a way to do it as any!